tests/stubtest_whitelists/py3_common.txt

```diff
@@ -12,6 +12,7 @@ _collections_abc.Mapping.get  # Adding None to the Union messed up mypy
 _collections_abc.Sequence.index  # Supporting None in end is not mandatory
 _csv.Dialect.__init__  # C __init__ signature is inaccurate
 _dummy_threading
+_socket.*
```

Comment:
Why is this needed?

Reply:
See above for the problems with the star imports. Since the star imports are now gone, this isn't needed anymore. Removing.